db/migrate/20201202161551_drop_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropTable < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :abbreviated_treatment_reports
+  end
+end
```

Comment:
@harithavytla One more change.  I don't think we need this migration file.  We don't need to drop the abbreviated_treatment_reports table.